sw/host/tests/manuf/cp_provision_init/src/main.rs

```diff
@@ -90,24 +93,36 @@ fn load_and_run_sram_program(opts: &Opts, transport: &TransportWrapper) -> Resul
     uart.clear_rx_buffer()?;
 
     // Load and execute the SRAM program that contains the provisioning code.
-    opts.sram_program
-        .load_and_execute(&jtag, ExecutionMode::Jump)?;
+    match opts
+        .sram_program
+        .load_and_execute(&jtag, ExecutionMode::Jump)?
+    {
+        ExecutionResult::Executing => {
+            log::info!("SRAM program loaded and is executing.")
+        }
+        res => log::info!("SRAM program load/execution failed: {:?}.", res),
+    }
 
     // Get UART, set flow control, and wait for test to start running.
     let uart = transport.uart("console")?;
     uart.set_flow_control(true)?;
     let _ = UartConsole::wait_for(&*uart, r"CP provisioning start.", opts.timeout)?;
-    let _ = UartConsole::wait_for(&*uart, r"CP provisioning end.", opts.timeout)?;
 
     // Inject provisioning data into the device.
-    let _provisioning_data = ManufCpProvisioningData {
+    // TODO(#19453): update provisioning_data sent to the device.
+    let provisioning_data = ManufCpProvisioningData {
         device_id: ArrayVec::from([0u32, 0u32, 0u32, 0u32]),
         manuf_state: ArrayVec::from([0u32, 0u32, 0u32, 0u32]),
         wafer_auth_secret: ArrayVec::from([0u32, 0u32, 0u32, 0u32]),
         test_unlock_token: ArrayVec::from([0u32, 0u32, 0u32, 0u32]),
         test_exit_token: ArrayVec::from([0u32, 0u32, 0u32, 0u32]),
```

Comment:
Should we make these into command line arguments or content from a config file?

Reply:
Yes I am working on that in a follow up currently.